### PR TITLE
Query uniswap trades against the new subgraph

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

**Comment by yabirgb** [comment]
I've updated the test and manually checked in ehterscan every transactions. I've found some strange thing like this transaction https://etherscan.io/tx/0xb41eeed3fde0b6376bba43523d1f8e18127587ffa269b343b7155805bda27270 where the graph returns this info

```       "swaps":[
          {
             "tx_hash":"0xb41eeed3fde0b6376bba43523d1f8e18127587ffa269b343b7155805bda27270",
             "log_index":124,
             "from_address":"0xc3037b2A1a9E9268025FF6d45Fe7095436446D52",
             "to_address":"0xc3037b2A1a9E9268025FF6d45Fe7095436446D52",
             "token0":"DAI",
             "token1":"WETH",
             "amount0_in":"226.067244853663419909",
             "amount1_in":"0",
             "amount0_out":"0",
             "amount1_out":"0.947308435652157362"
          },
          {
             "tx_hash":"0xb41eeed3fde0b6376bba43523d1f8e18127587ffa269b343b7155805bda27270",
             "log_index":128,
             "from_address":"0xc3037b2A1a9E9268025FF6d45Fe7095436446D52",
             "to_address":"0xc3037b2A1a9E9268025FF6d45Fe7095436446D52",
             "token0":"LEND",
             "token1":"WETH",
             "amount0_in":"0",
             "amount1_in":"0.947308435652157362",
             "amount0_out":"2340.535105624007589249",
             "amount1_out":"0"
          }
       ]```

and the second swap should have 

```            
 "token0":"WETH",
 "token1":"LEND",
```
I believe. I've checked this and the official graph returns the same result.

**Comment by LefterisJP** [comment]
> and the second swap should have
> 
> ```
>  "token0":"WETH",
>  "token1":"LEND",
> ```

Why do you believe the second swap should have this order?

The corresponding pool is this: https://etherscan.io/address/0xab3f9bf1d81ddb224a2014e98b238638824bcf20#readContract



So the `token0` and `token1` look correct to me. Token0/Token1 is pool properties, not swap properties. You can see that in is the token1 amount and out is the token0 amount, which do agree with the order shown in etherscan.

**Comment by yabirgb** [comment]
@LefterisJP Okey then it was a misunderstand on my side

File: rotkehlchen/tests/api/test_uniswap.py [hunk 1]
```diff
@@ -156,244 +156,302 @@ def test_get_balances(
...
         amount0_in=AssetAmount(ZERO),
-        amount1_in=AssetAmount(FVal('54.970103483445793496')),
-        amount0_out=AssetAmount(FVal('49.398809059381378894')),
+        amount1_in=AssetAmount(FVal('12.605060360501788046')),
+        amount0_out=AssetAmount(FVal('21928.266845570798012768')),
         amount1_out=AssetAmount(ZERO),
     )],
 ), AMMTrade(
     trade_type=TradeType.BUY,
-    base_asset=EthereumToken('yyDAI+yUSDC+yUSDT+yTUSD'),
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_uniswap.py:305]: Oh man all these will make me suffer in the rebase of my PR on top of develop.

**Review by LefterisJP — APPROVED** [review]
It will suck to rebase on top of this unfortunately for me.

But the PR looks good. Also @coinyon confirmed that it solved the missing trades for him so we will assume that switching to this other subgraph solves it for most people.

According to @yabirgb switching to this subgraph also solves https://github.com/Uniswap/uniswap-v2-subgraph/issues/85 so it starts to be a no brainer.

**Pull request merged**